File: class10/java/AP1005-BoletimMatriz.java
public class BoletimMatriz {
    public static void main(String[] args) {
        final int ALUNOS = 3, BIM = 4;
        double[][] notas = {
            {7.5, 8.0, 6.5, 9.0},
            {5.0, 6.0, 7.0, 8.5},
            {9.0, 9.5, 8.0, 7.5}
        };
        for (int i = 0; i < ALUNOS; i++) {
            double soma = 0;
            for (int j = 0; j < BIM; j++) soma += notas[i][j];
            System.out.printf("Media do aluno %d: %.2f%n", i, soma / BIM);
        }
    }
}

File: class10/python/AP1005-BoletimMatriz.py
# Boletim: matriz alunos x bimestres, media por aluno (loops aninhados)
ALUNOS, BIM = 3, 4
notas = [[7.5, 8.0, 6.5, 9.0],
         [5.0, 6.0, 7.0, 8.5],
         [9.0, 9.5, 8.0, 7.5]]
for i in range(ALUNOS):
    soma = 0.0
    for j in range(BIM):
        soma += notas[i][j]
    media = soma / BIM
    print(f"Media do aluno {i}: {media:.2f}")
